service/header/p2p_exchange.go

```diff
@@ -58,111 +57,20 @@ func (ex *P2PExchange) Start(ctx context.Context) error {
 
 		err := ex.host.Connect(ctx, *ex.trustedPeer)
 		if err != nil {
-			log.Errorw("connecting to trusted peer", "err", err)
-			log.Warn("HEADERS WONT BE SYNCHRONIZED - PLEASE RESTART WITH TRUSTED PEER BEING ONLINE")
+			log.Errorw("p2p: connecting to trusted peer", "err", err)
+			log.Warn("p2p: HEADERS WONT BE SYNCHRONIZED - PLEASE RESTART WITH TRUSTED PEER BEING ONLINE")
```

Comment:
orthogonal to this PR but please remind me: so in case we come online without a trusted peer, the node does not really do anything because it won't sync headers?

Reply:
For devnet we defined its is acceptable to do so, but it should change so we can download from any peer and trusted peer will only stay for subjective initialization.

Reply:
I hope this is also tracked in an issue. Resolving now.